sevntu-checks/src/main/java/com/github/sevntu/checkstyle/Utils.java

```diff
@@ -74,4 +77,63 @@ public static DetailAST getNextSubTreeNode(DetailAST node, DetailAST subTreeRoot
         }
         return toVisitAst;
     }
+
+    /**
+     * Obtain full type name of first token in node.
+     * @param ast an AST node
+     * @return fully qualified name (FQN) of type, or null if none was found
+     */
+    public static String getTypeNameOfFirstToken(final DetailAST ast) {
+        final DetailAST findFirstToken = ast.findFirstToken(TokenTypes.TYPE);
+        if (findFirstToken != null) {
+            final FullIdent ident = CheckUtils.createFullType(findFirstToken);
+            if (ident != null) {
+                return ident.getText();
```

Comment:
It is already an open issue in main project where sevntu get its configuration from.
Just no one has worked on it.